benchmarks/transform.py

```diff
@@ -0,0 +1,13 @@
+from geopandas import GeoDataFrame, read_file, datasets
+import pandas as pd
+
+
+class Boroughs:
```

Comment:
Can you give this a name on what it is about? Eg `CRS`

Reply:
Sure, just did that. In looking at the names of the classes in the other benchmarks, the pattern was a little opaque to me.